faq.md

```diff
@@ -17,6 +38,12 @@ Please read the installation instructions for the appropriate distribution:
 - [Debian installation instructions](running.html#debian)
 - [Ubuntu installation instructions](running.html#ubuntu)
 
+### How do I use Cockpit on another port?
+
+Cockpit runs on port 9090 by default. If you'd like to change the port to something else, please [consult the Cockpit guide's section on TCP port and address](https://cockpit-project.org/guide/latest/listen).
+
+Additionally, it's possible to stop the Cockpit socket systemd service and log into Cockpit by SSH only. For this to work, you have to use a Linux-based desktop and run [the Cockpit Client desktop application, available on Flathub](https://flathub.org/apps/details/org.cockpit_project.CockpitClient).
```

Comment:
This feels a little misplaced here. I'd make this another section, like "How do I run Cockpit without opening a new port?". We can then also mention the bastion mode of the cockpit/ws container.

Reply:
Sure. Please feel free to split it off.

(I'd rather not block the FAQ updates too long, one way or another.)

Reply:
I split it out into a new paragraph. WDYT?